synfig-studio/src/gui/trees/layertreestore.h

```diff
@@ -85,7 +85,7 @@ class LayerTreeStore : public Gtk::TreeStore
 
 		Gtk::TreeModelColumn<bool>						children_lock;
 
-		Gtk::TreeModelColumn<float> z_depth;
```

Comment:
Better to use `Gtk::TreeViewColumn::set_cell_data_func` to change only column presentation, not value.
https://developer-old.gnome.org/gtkmm/3.24/classGtk_1_1TreeViewColumn.html#a3469e1adf42e5932ea123ec33e4ce4e1

Reply:
sounds like a safer approach, ok I'm looking into how to implement it.

Reply:
done